Sourcery/Generating/Template/Stencil/StencilTemplate.swift

```diff
@@ -85,6 +92,15 @@ extension Annotated {
 }
 
 extension Stencil.Extension {
+    
+    func registerFilterWithTwoArguments<A, B>(_ name: String, filter: @escaping (Any?, A, B) throws -> Any?) {
+        registerFilter(name) { (any, args) throws -> Any? in
+            guard args.count == 3, let argA = args[0] as? A, let argB = args[2] as? B else {
+                throw TemplateSyntaxError("'\(name)' filter takes two arguments: \(A.self) and \(B.self)")
+            }
+            return try filter(any, argA, argB)
+        }
+    }
```

Comment:
I'm also happy to make changes, as desired. I tried to follow suit by wrapping this in a `registerStringFilterWithTwoArguments`, but I couldn't quite get Swift compiler errors, so I left it as-is. I also wasn't sure what's the best way to fail